.changeset/many-lobsters-accept.md

```diff
@@ -0,0 +1,8 @@
+---
+"astro": minor
```

Comment:
What's the reason of a `minor`? Should this be a patch?

Reply:
I always prefer patch level ;-), but here I am unsure whether the fix should be a minor as it might break code in rare cases. Astro users who only use `transition:name` will not notice any negative effects. In niche cases there may be surprises, e.g. if you use `transition:name` in an .astro file and `view-transition-name` CSS property in an .mdx file and both should match. Then it may be that they differ after we have changed the coding scheme. 

As I just wanted to construct an example that will a) differ and has b) worked before this fix, I see that those cases might be really rare. One would be "a_b" which now encodes as "a__b".

What do you think? Should I revert to "patch"?

Reply:
I see! We can go with a `minor` then. Considering your comment, I believe the changelog should highlight the possibility of a "breaking change", meaning that users who were "leveraging the bug" might be impacted.

Reply:
Minor is ok in this case since the release is only 2 days from now. If it were next week I'd rather fix the known regression than worry about unknown corner cases. But 2 days is ok.